attributes/customized-multipath.rb

```diff
@@ -0,0 +1,4 @@
+if node['multipath']['storage_type'] == "customized"
```

Comment:
The logic was left-over from a previous iteration I had and I forgot to remove it.  I think it would suffice to just remove the logic, don't you?